tests/rip7560/test_rip7560.py

```diff
@@ -50,10 +78,12 @@ def test_getTransaction(w3, wallet_contract, tx_7560):
         type=4,
         blockHash=block["hash"],
         blockNumber=block["number"],
-        gasPrice=min(int(tx_7560.maxFeePerGas, 16), block['baseFeePerGas']+int(tx_7560.maxPriorityFeePerGas, 16)),
+        gasPrice=min(
```

Comment:
I don't mind these reformattings.. but I think it would be best to do a "refactor" pr, instead of scatter them around inside technical PRs.

Reply:
I don't do it manually, I'm running `pdm run format` in a git hook, and we're using Black as a formatter.

Reply:
no problem. that's great.
I just suggest to initially run it in a separate PR, and then we all run it as a git hook.
(afaik, goland also has auto-formatting, but different, so we need to align them. with "typescript" we could simply import "prettier" config into the ide, so the separate prettier step wouldn't have to do much, just verify.